test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -60,7 +60,20 @@ export default class PerfCompare extends Component {
     };
 
     componentDidMount() {
+        const currentURL = window.location.href;
 
+        const path = currentURL.split('?')[1];
+        if (path) {
+            const state_vars = path.split("&");
+            for (const item in state_vars) {
+                const key = state_vars[item].split("=")[0];
+                const value = state_vars[item].split("=")[1];
+
+                this.state.inputURL[key] = value;
+            }
+        }
```

Comment:
- instead of directly setting the value in the state, please use setState() 

- We should use the existing function `getParams()` and it will return an object of key and value. For example, 

```
getParams(window.location.search)
```

Reply:
Nice suggestion! We'll do the same in `TabularView.jsx` as well. Thanks!